lang/en.lyaml

```diff
@@ -1517,6 +1517,10 @@ en:
         inactiveSecondary: "Run {{ command }} to remove inactive accounts from your CLI config"
         unableToDetermine: "Unable to determine if the portal is active"
         pak:
+          incomplete:
+            one: "Personal access key is valid. {{ count }} scope available, but not included in your key."
```

Comment:
What are your thoughts on something like this for copy?
"Personal access key is valid, but you have 11 available scopes that are not included in your key"

This might just be me, but "N scopes available, but not included in your key." reads a little awkwardly. I initially read it as "you have 11 scopes total"

Reply:
Agreed, I reworded after removing the count and it's a lot smoother IMO